src/codegen/translate-instance.lisp

```diff
@@ -107,7 +131,7 @@
                app-node))
 
          (bindings
-           (cons
+           (list*
             (cons
              (tc:ty-class-instance-codegen-sym instance-entry)
              dict-node)
```

Comment:
Disappointed that we don't see `ACONS` here

Reply:
good call

Reply:
fixed